app/src/main/java/solendsdkfacade/SolendMethodResultHandler.kt

```diff
@@ -0,0 +1,9 @@
+package solendsdkfacade
+
+import solendsdkfacade.model.SolendMethodResultError
+import solendsdkfacade.model.SolendMethodResultSuccess
+
+interface SolendMethodResultHandler {
```

Comment:
Handler in future - is a foreground service / workmanager that handles async result from SDK and works with UI to show toasts and stuff